src/appmixer/slack/routes-tasks.js

```
@@ -176,9 +141,58 @@ module.exports = (context) => {
                 }
 
                 const { actions, response_url: responseUrl } = payload;
-                const taskId = actions[0].value;
+                if (!actions || !Array.isArray(actions) || actions.length === 0) {
+                    return h.response({ text: 'No actions found' }).code(400);
+                }
+                // value format: taskId|host, e.g. "taskId|https://api.acme.appmixer.cloud"
+                const [taskId, host] = actions[0].value?.split('|');
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)